src/Features/Core/Portable/GoToDefinition/AbstractGoToDefinitionSymbolService.cs

```diff
@@ -31,6 +35,16 @@ public async Task<(ISymbol?, TextSpan)> GetSymbolAndBoundSpanAsync(Document docu
             return (FindRelatedExplicitlyDeclaredSymbol(symbol, semanticModel.Compilation), semanticInfo.Span);
         }
 
+        public async Task<int?> GetTargetIfControlFlowAsync(Document document, int position, CancellationToken cancellationToken)
+        {
+            var syntaxTree = await document.GetRequiredSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
+            var language = syntaxTree.GetRoot(cancellationToken).Language;
```

Comment:
```suggestion
```

Reply:
Not sure what this is referring to

---
In reply to: [589709698](https://github.com/dotnet/roslyn/pull/51063#discussion_r589709698) [](ancestors = 589709698)

Reply:
Never mind. Figure it out (it's unused)

---
In reply to: [589952359](https://github.com/dotnet/roslyn/pull/51063#discussion_r589952359) [](ancestors = 589952359,589709698)

Reply:
wat happened here...